### PR TITLE
nav link

### PR DESCRIPTION
**Pull request by AfamsVal**
Please review. Thanks!

### CODE REVIEW

**Review by uimarshall — COMMENTED** [review]
On what basis are the prices changing on toggling? Is the price changes based on the number of orders?

**Review by EngrEric — APPROVED** [review]

File: src/components/Pricing/PricingPro.js [hunk 1]
```diff
@@ -8,8 +8,24 @@ import PriceCardFour from "./pricing-pro-cards/PriceCardFour";
...
   const [price, setPrice] = useState({});
 
-  const monthly = { type: "monthly", pro100: 29, pro500: 79, pro2k: 179 };
-  const annual = { type: "annual", pro100: 24, pro500: 66, pro2k: 149 };
+  const monthly = {
+    type: "monthly",
+    pro100: 29,
+    pro500: 79,
+    pro2k: 179,
+    adsPro100: 48,
```
Comment (uimarshall) [inline src/components/Pricing/PricingPro.js:15]: 16 + 29 should not give you **48** on toggle as addOns but **45** compare [sellbrite](https://www.sellbrite.com/pricing-pro/) which has 19 + 29 = 48 on toggle.

File: src/components/Pricing/PricingPro.js [hunk 2]
```diff
@@ -8,8 +8,24 @@ import PriceCardFour from "./pricing-pro-cards/PriceCardFour";
...
+    adsPro500: 98,
```
Comment (uimarshall) [inline src/components/Pricing/PricingPro.js:16]: 16 + 79 should not give you 98 on toggle as addOns but 95

File: src/components/Pricing/PricingPro.js [hunk 3]
```diff
@@ -8,8 +8,24 @@ import PriceCardFour from "./pricing-pro-cards/PriceCardFour";
...
+    adsPro2k: 198,
```
Comment (uimarshall) [inline src/components/Pricing/PricingPro.js:17]: You should have 195 on toggle.

**Review by uimarshall — CHANGES_REQUESTED** [review]

**Review by uimarshall — APPROVED** [review]
Looks good, you can merge 👍

**Pull request merged**